webapp/src/components/properties/multiSelect.tsx

```diff
@@ -25,9 +25,7 @@ const MultiSelectProperty = (props: Props): JSX.Element => {
     const {propertyTemplate, emptyValue, propertyValue, isEditable, onChange, onChangeColor, onDeleteOption, onCreate, onDeleteValue} = props
     const [open, setOpen] = useState(false)
 
-    const values = Array.isArray(propertyValue) ?
-        propertyValue.map((v) => propertyTemplate.options.find((o) => o!.id === v)).filter((v): v is IPropertyOption => Boolean(v)) :
-        []
+    const values = Array.isArray(propertyValue) && propertyValue.length > 0 ? propertyValue.map((v) => propertyTemplate.options.find((o) => o!.id === v)).filter((v): v is IPropertyOption => Boolean(v)) : []
```

Comment:
Is there a limit for the line length for the project code style?

Reply:
Not exactly sure what you're asking. We currently don't have a limit on either the line length nor the number of property options, which this line is dealing with.

Reply:
Was asking about the line length for the source code.